prisma/schema.prisma

```diff
@@ -425,7 +425,6 @@ model mentor {
   lastname                     String?                 @db.VarChar
```

Comment:
Also, undo the changes to this file. The idea is that we first stop using the verification column in favor of `verifiedAt`.
The part of dropping the verification column from the DB should be done in a separate pull request :)

Reply:
Here there are also changes with the ordering / formatting. Could you also fix those?